### PR TITLE
Support qbft encoding of extra data in the operator generate-blockchain subcommand

### LINKED ISSUE #2049: QBFT extra data subcommand
Add a QBFT extra data subcommand so that users can create the necessary extra data for the genesis file.

### PR DESCRIPTION
**Pull request by jframe**
Signed-off-by: Jason Frame <jasonwframe@gmail.com>

## PR description
Support qbft encoding of extra data in the operator generate-blockchain subcommand

## Fixed Issue(s)

fixes #2049

## Changelog

- [x] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

File: besu/src/main/java/org/hyperledger/besu/cli/subcommands/operator/GenerateBlockchainConfig.java [hunk 1]
```diff
@@ -242,6 +243,11 @@ private void processExtraData() {
       final String extraData =
           IbftExtraDataCodec.encodeFromAddresses(addressesForGenesisExtraData).toString();
       genesisConfig.put("extraData", extraData);
+    } else if (genesisConfigOptions.isQbft()) {
```
Comment (usmansaleem) [inline besu/src/main/java/org/hyperledger/besu/cli/subcommands/operator/GenerateBlockchainConfig.java:246]: on line 239, should a check for null be introduced for `configNode` before continuing?

**Review by usmansaleem — APPROVED** [review]
LGTM - one minor comment for potential NPE check.

File: besu/src/main/java/org/hyperledger/besu/cli/subcommands/operator/GenerateBlockchainConfig.java (hunk 1 above)
Comment (jframe) [reply besu/src/main/java/org/hyperledger/besu/cli/subcommands/operator/GenerateBlockchainConfig.java:246]: Good catch. I'll add a check for that.

**Review by usmansaleem — APPROVED** [review]
LGTM

**Pull request merged**